### PR TITLE
Added via support for 7sKB

### PR DESCRIPTION
**Pull request by Salicylic-acid3**
-Changing the VID
-Add a keymap via

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/7skb/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#include QMK_KEYBOARD_H
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
```
Comment (zvecr) [inline keyboards/7skb/keymaps/via/keymap.c:9]:
```suggestion

```

File: keyboards/7skb/rev1/config.h [hunk 2]
```diff
@@ -19,7 +19,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x5333
```
Comment (fauxpark) [inline keyboards/7skb/rev1/config.h:22]:
```suggestion
#define VENDOR_ID       0x5333 // "S3"
```

**Comment by Salicylic-acid3** [comment]
Sorry.
I changed it because I got a new VID.

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**